### PR TITLE
[AUD-937 AUD-938 AUD-939] Add NFT option to image picker

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
[AUD-937 AUD-938 AUD-939]

### Description

- Added NFT option to the ImageSelectionPopup behind a feature flag
- Added a button to set profile picture to collectible details modal
- Updated TabSlider to work better with flexible width and not have a hard-coded padding value

### How Has This Been Tested?

Lots of manual testing

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/kj-Add-NFT-to-image-picker

**Comment by raymondjacobson** [comment]
frontend demo has something wrong with it 💀  wanna push to a subdomain to play with?

**Comment by Kyle-Shanks** [comment]
Demo on kj.audius.co 👍

File: src/components/image-selection/ImageSelectionPopup.js [hunk 1]
```diff
@@ -114,6 +121,75 @@ const RandomPage = ({ onSelect }) => {
...
+  const { isEnabled: isSolanaCollectiblesEnabled } = useFlag(
+    FeatureFlags.SOLANA_COLLECTIBLES_ENABLED
+  )
+  const { collectibleList, solanaCollectibleList } = useSelector(getAccountUser)
+  const allCollectibles = [
+    ...(collectibleList || []),
+    ...((isSolanaCollectiblesEnabled && solanaCollectibleList) || [])
+  ]
+  const [page, setPage] = useState(1)
+  const numPerPage = 15
```
Comment (raymondjacobson) [inline src/components/image-selection/ImageSelectionPopup.js]: maybe should be a const at the top of the file?

File: src/containers/collectibles/components/CollectibleDetails.tsx [hunk 2]
```diff
@@ -383,6 +410,58 @@ const CollectibleDetails: React.FC<{
                 {collectibleMessages.linkToCollectible}
               </a>
             )}
+
+            {collectible.mediaType === CollectibleMediaType.IMAGE && (
+              <Button
```
Comment (raymondjacobson) [inline src/containers/collectibles/components/CollectibleDetails.tsx:415]: need to wrap this in the feature flag

**Review by raymondjacobson — APPROVED** [review]
1 comment about feature flag

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/kj-Add-NFT-to-image-picker

File: src/components/data-entry/TabSlider.js [hunk 3]
```diff
@@ -6,9 +6,6 @@ import { useSpring, animated } from 'react-spring'
```
[the remark this answers is not in the document]
Comment (Kyle-Shanks) [reply src/components/data-entry/TabSlider.js:6]: Will update TabSlider in stems and update all instance of it in audius-client in a separate PR 👍

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/kj-Add-NFT-to-image-picker

**Pull request merged**